### DISCUSSION TITLE
Azure Deployment / Amend "wiki/server/master.js"

### DISCUSSION THREAD

**Discussion by ThatRobbo** [requarks/wiki#4972] [Help / Questions]
Hi,
Really new deployment and I've come across the issue with app.use(bodyParser.json({ limit: '1mb' })) when saving pages with large content or draw.io documents. I've found the articles about this, but I've deployed it within Azure so I'm not entirely sure how to access the maser.js within Azure. I've tried using FTP but can't see the file in question. Can anyone help me find how to access this and amend the file? Thanks in advance

**Comment by jtrencan** [comment]
i had problem to see the files (pictures and documents in "assets library") when i was connecting with ftp connection. i had to adjust settings for ftp ans SELinux (i am using rocky linux 9.4) 
i had used these command to setup SELinux properties (they are copied from copilot) 

sudo dnf install policycoreutils policycoreutils-python-utils
sudo setsebool -P ftpd_full_access 1
sudo setsebool -P ftp_home_dir 1
sudo systemctl restart vsftpd

so i am able to see files on server and add them, but i am not able see uploaded files from "editors interface" on the wiki page. thus i was able to upload them, but i am not able to use them.
